packages/usa-layout-grid/src/styles/_usa-layout-grid.scss

```
@@ -6,6 +6,9 @@
 
 $namespace-grid: ns("grid");
 
+$custom-breakpoints: map-deep-get($system-properties, breakpoints, extended);
+$all-breakpoints: map-collect($system-breakpoints, $custom-breakpoints);
```

>[!note]
> Interestingly, layout grid and icon list size use [$system-breakpoints in breakpoints.scss](https://github.com/uswds/uswds/blob/develop/packages/uswds-core/src/styles/tokens/units/breakpoints.scss) and the other utilities build from [`$system-properties.breakpoints` in _properties.scss](https://github.com/uswds/uswds/blob/develop/packages/uswds-core/src/styles/_properties.scss#L157). Both seem to collect the same values:

```scss
/*breakpoints.scss*/
$system-breakpoints: general.map-collect(
  map.get(spacing.$system-spacing, "large"),
  map.get(spacing.$system-spacing, "larger"),
  map.get(spacing.$system-spacing, "largest")
);
```

```scss
/*_properties.scss*/
  breakpoints: (
    standard:
      map-collect(
        map.get($system-spacing, "large"),
        map.get($system-spacing, "larger"),
        map.get($system-spacing, "largest")
      ),
    extended: (),
  ),
```

Flagging in case we want to standardize this and potentially replace `$system-breakpoints` with `$system-properties, breakpoints, standard` here. To keep things scoped, I am leaving `$system-breakpoints` as is for now.

---

@amyleadem good find. Do you have an initial recommendation?

---

We could remove the reference to `$system-breakpoints` and instead use the [same technique from _utility-builder.scss](https://github.com/uswds/uswds/blob/develop/packages/uswds-core/src/styles/mixins/_utility-builder.scss#L356) to grab `$our-breakpoints` from `$system-properties`. Initial testing on this branch shows that this change creates the correct number of references in the CSS, but it would need further testing to confirm that the generated CSS is correct. What do you think - should we pursue now or explore as a possible follow-up?


```suggestion
$our-breakpoints: map-deep-get($system-properties, breakpoints, standard);
$all-breakpoints: map-collect($our-breakpoints, $custom-breakpoints);
```

---

@amyleadem inclined to say follow-up so as to not delay the release.